### DISCUSSION TITLE
How to fix this😔

### DISCUSSION THREAD

**Discussion by unknownMaskara** [jaykali/maskphish#24] [Q&A] [answered]
![IMG_20210312_141236](https://user-images.githubusercontent.com/58129684/110900047-1bf1b080-833d-11eb-8148-ca0f85a70f0e.jpg)

It ends with @ or @blacklisted
how to fix?

**Comment by technicaldada** [comment]
You have put wrong phishing URL there. The phishing link will be http://d06b8e***.ngrok.io.

You already put there a masked URL.

**Comment by unknownMaskara** [comment]
![IMG_20210312_162925](https://user-images.githubusercontent.com/58129684/110913539-44cf7100-8350-11eb-8d35-e8a4c31d5b35.jpg)

😔😔still the same. i used Maskphish v. 2.0

**Comment by technicaldada** [comment]
Check we have did the same thing
![Screenshot_20210312-171447](https://user-images.githubusercontent.com/32122182/110936214-8bc06500-8356-11eb-85a7-37cd5ab09bc5.png)
Everything is perfect here. Are you sure that you are using v2 of MaskPhish and didn't use space after ngrok URL.

**Reply by unknownMaskara** [reply]
not sure if i installed v.2.0 sir...what is the command to use..or where i can get the v. 2.0?

**Reply by unknownMaskara** [reply]
How to install v. 2.0?

**Reply by rayquazza49** [reply]
im having the same issue still not solved

**Reply by rayquazza49** [reply]
![Screenshot from 2021-08-08 21-27-23](https://user-images.githubusercontent.com/78814455/128638065-0eafdbf4-4cc4-424d-9b30-ad3d5b5a8883.png)

**Reply by jaykali** [reply]
> ![Screenshot from 2021-08-08 21-27-23](https://user-images.githubusercontent.com/78814455/128638065-0eafdbf4-4cc4-424d-9b30-ad3d5b5a8883.png)

Seems you cant access [https://is.gd](https://is.gd) services. We use that URL shortner to short the Maskphish URL. Please check if the website is accessible from your web browser. If not then please use VPN. Untill we release V3 of MaskPhish.

**Comment by jaykali** [comment, marked as answer]
Hi @unknownMaskara ,
Welcome to discussion of MaskPhish.

Giving commands to uninstall MaskPhish and install the newer version. Open Termux and apply following commands:
`rm -rf maskphish`
`git clone https://github.com/jaykali/maskphish`
`cd maskphish`
`./maskphish.sh`
@technicaldada Thanks a lot for helping others. You are right we should not make any mistake or extra space on the phishing URL field.
@unknownMaskara Hope this will help you.

**Reply by Anhedoniczz** [reply]
i tried it and got same error can you help me ?
![Screenshot at 2021-10-23 20-54-34](https://user-images.githubusercontent.com/93049946/138565107-527d0f37-e5d2-4a61-bb15-969f1d7a745c.png)

**Reply by jaykali** [reply]
https://is.gd server is down. We uses that services to short urls. Can you please wait for a week? We are going to update it.

**Comment by unknownMaskara** [comment]
![IMG_20210313_214850](https://user-images.githubusercontent.com/58129684/111032198-fdc3a780-8445-11eb-8ba7-f6c98647d4d7.jpg)

Thanks sir..but bash: ./maskphish: No such file or directory 😔

**Reply by technicaldada** [reply]
@jaykali  did typing mistake there. It will be
`./maskphish.sh`
@unknownMaskara didn't you have basic Linux knowledge?

**Reply by jaykali** [reply]
You are right @technicaldada That was my mistake. Thanks

**Reply by unknownMaskara** [reply]
Thanks!!! Its working in Kali😊But phishing link not working in Chrome browser mobile..But its working in other browser like Uc browser or firefox, etc.

**Reply by jaykali** [reply]
This also works on Chrome but Chrome shows a warning there. The browser got updated and shows the warning.

**Comment by Kartik-spec02** [comment]
There's a small error with maskphish, the created link ends with @ in kali linux could you guys help me with the solution?

**Comment by profesionalleet1337** [comment]
I also have problem with setting up the link's which are generated by the tool ZPISHER. They  gone detected and deleted ? { Sorry for my grammar, hope for advice/ sollution about this : ) }
